en/docs/guides/identity-lifecycles/outbound-provisioning-with-salesforce.md

```diff
@@ -157,31 +157,292 @@ too.
     2.  Click **Create Self-Signed Certificate**.
     3.  Enter the **Label** and a **Unique Name** and click **Save**.
         The certificate is generated.  
-        ![self-signed-certificate](../../assets/img/guides/add-certificate.png) 
+        ![self-signed-certificate](../../../assets/img/guides/add-certificate.png) 
     4.  Click the **Download Certificate** button to download the
         certificate.
 
 ---
 
 ## Configure email address as the username
 
-(TODO:insert-admin-portal)
+Provisioning is the process of coordinating the creation of user
+accounts, e-mail authorizations in the form of rules and roles, and
+other tasks such as provisioning of resources associated with enabling
+new users.
+
+1.  Download the WSO2 Identity Server from [here](http://wso2.com/products/identity-server/) and [run
+    it](../../../deploy/get-started/run-the-product/).
+2.  Log in to the Management Console (`https://<IS_HOST>:<PORT>/carbon`) as an
+    administrator.
+3.  When you log into Salesforce, you normally use an email address. So,
+    to integrate this with the Identity Server, you need to configure
+    WSO2 IS to enable users to log in using their email addresses. In
+    order to do that, follow the steps found in the [Configure Email Address as the Username](../../../guides/identity-lifecycles/enable-email-as-username) topic.
```

Comment:
since this is a mandatory step shouldn't we have the steps inline? there is a fragment for this